### PR TITLE
Added/Updated packages, descriptions

### PR DESCRIPTION
**Pull request by Blitz517**
added more packages and used from old pull requests add/changes packages, descriptions.

### CODE REVIEW

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -36598,5 +36597,572 @@
...
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.huawei.dmsdp",
+    "list": "Oem",
+    "description": "Multi-Device management\nThis service allows you to virtually expand your devices capabilities\nby connecting to other devices and using them as an extension of this device.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:36896]:
```suggestion
    "removal": "Advanced"
```
Some users may want to use this.

File: resources/assets/uad_lists.json [hunk 2]
```diff
@@ -36598,5 +36597,572 @@
...
+    "list": "Oem",
+    "description": "MediaLibrary\nhas hwddmp components `com.huawei.hwddmp` debuging errors, remote device manager that depends on Huawei account",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.android.systemui.overlay",
+    "list": "Oem",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:37107]:
```suggestion
    "list": "AOSP",
```

File: resources/assets/uad_lists.json [hunk 3]
```diff
@@ -36598,5 +36597,572 @@
...
+    "description": "waudio\nWi-Fi Speaker, Speaker playback, WLAN speaker\nThis app is only for Chinese users because I found a Chinese app list that supports it\nand depends on AI Life app probably.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "ohos.media.medialibrary",
+    "list": "Oem",
+    "description": "MediaLibrary\nhas hwddmp components `com.huawei.hwddmp` debuging errors, remote device manager that depends on Huawei account",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:37099]:
```suggestion
    "description": "MediaLibrary\nhas hwddmp components `com.huawei.hwddmp` debugging errors, remote device manager that depends on Huawei account",
```

File: resources/assets/uad_lists.json [hunk 4]
```diff
@@ -36598,5 +36597,572 @@
...
+    "description": "OneHopSvcClient\nTagService, has only tracking things, hmskit that does not exist on the site of of Huawei.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.huawei.onehopsvchost",
+    "list": "Oem",
+    "description": "OneHopSvcHost\nTriggerservice, has only tracking things, hmskit that does not exist on the site of of Huawei.",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:37018]:
```suggestion
    "description": "OneHopSvcHost\nTriggerservice, has only tracking things, hmskit that does not exist on the site of Huawei.",
```

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

File: resources/assets/uad_lists.json [hunk 5]
```diff
@@ -14537,7 +14537,7 @@
   {
     "id": "com.qualcomm.qtil.btdsda",
```
[the remark this answers is not in the document]
Comment (Blitz517) [reply resources/assets/uad_lists.json:14538]: App size is too small

**Review by AnonymousWP — APPROVED** [review]

File: resources/assets/uad_lists.json [hunk 6]
```diff
@@ -36598,5 +36597,572 @@
...
+    "id": "com.android.systemui.accessibility.accessibilitymenu",
+    "list": "Aosp",
+    "description": "Hidden menu that only shows 2 buttons:\nLarge buttons - that increases size of accessibility menu buttons,\nand Help - that redirects to support google com site accessibility.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "org.lineageos.profiles",
```
Comment (Rudxain) [inline resources/assets/uad_lists.json:36728]: Why were these added? This should be documented by Lineage, not UAD

File: resources/assets/uad_lists.json [hunk 7]
```diff
@@ -36598,5 +36597,572 @@
...
+  },
+  {
+    "id": "org.lineageos.setupwizard",
+    "list": "Oem",
+    "description": "Setup Wizard\nNeeded only on first-boot setup.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
```
Comment (Rudxain) [inline resources/assets/uad_lists.json:36753]: Why were these added? This should be documented by Lineage, not UAD

Comment (Rudxain) [reply resources/assets/uad_lists.json:36753]: #812

**Pull request merged**